include/openmc/mesh.h

```diff
@@ -715,7 +717,8 @@ class LibMesh : public UnstructuredMesh {
   int get_bin_from_element(const libMesh::Elem* elem) const;
 
   // Data members
-  unique_ptr<libMesh::Mesh> m_; //!< pointer to the libMesh mesh instance
+  unique_ptr<libMesh::MeshBase> unique_m_; //!< pointer to the libMesh MeshBase instance, only used if mesh is created inside OpenMC
+  libMesh::MeshBase * m_; //!< pointer to libMesh MeshBase instance
```

Comment:
```suggestion
  libMesh::MeshBase * m_; //!< pointer to libMesh MeshBase instance, always set during intialization
```

Reply:
Should the accessor be a private or protected member function of LibMesh? @pshriwise

Reply:
Re: slack convo, I will make this accessor a public member function.

Reply:
Changed it to header one-liner